metrics-operator/config/webhook/manifests.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
the name of this file also needs to be added to the `resources` list of the `kustomization.yaml` in the same directory